rpc/cache/service.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.27.1
-// 	protoc        v3.19.4
+// 	protoc        v3.19.1
```

Comment:
We probably should update the version to 3.19.4.
https://github.com/aquasecurity/trivy/blob/3b0b2ed4ce481c7f058d47b0f128c906fb91a958/Dockerfile.protoc#L4